test/suite/test_timestamp08.py

```diff
@@ -105,7 +105,7 @@ def test_timestamp_api(self):
         self.session.timestamp_transaction_uint(wiredtiger.WT_TS_TXN_TYPE_COMMIT, 7)
         self.session.commit_transaction()
 
-        # Read timestamp >= oldest timestamp
+        # Read timestamp < oldest timestamp
```

Comment:
Good spot - thanks. I think this was wrong originally.

Reply:
[2c84acb](https://github.com/wiredtiger/wiredtiger/pull/10088/commits/2c84acb9ab2967d5f81bc1d18107430f965ce2fb)